Freshli.Web/Startup.cs

```diff
@@ -81,7 +81,7 @@ public class Startup {
           endpoints.MapControllers();
           endpoints.MapControllerRoute(
             name: "default",
-            pattern: "{controller=Home}/{action=Index}/{id?}"
+            pattern: "{controller=AnalysisRequests}/{action=Create}/{id?}"
```

Comment:
I wasn't a fan of having the urls read `~/{id}`, so I changed this back to `Home` in 38569c4.